controllers/admin/AdminStoresController.php

```diff
@@ -460,8 +460,8 @@ protected function _getDefaultFieldsContent()
                 'type' => 'text',
             ],
             'PS_SHOP_DETAILS' => [
-                'title' => $this->trans('Registration number', [], 'Admin.Shopparameters.Feature'),
-                'hint' => $this->trans('Shop registration information (e.g. SIRET or RCS).', [], 'Admin.Shopparameters.Help'),
+                'title' => $this->trans('Company details', [], 'Admin.Shopparameters.Feature'),
+                'hint' => $this->trans('Shop registration information (e.g. DNI, VAT number, SIRET or RCS).', [], 'Admin.Shopparameters.Help'),
```

Comment:
```suggestion
                'hint' => $this->trans('Company's registration details (e.g. DNI, VAT, SIRET, RCS, etc.).', [], 'Admin.Shopparameters.Help'),
```

The last comment before I can approve wording. :-)